source/extensions/certificate_providers/default_cert_provider/config.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "envoy/common/pure.h"
+#include "envoy/config/core/v3/extension.pb.h"
+#include "source/extensions/certificate_providers/factory.h"
+#include "source/extensions/certificate_providers/certificate_provider.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace CertificateProviders {
+
+class DefaultCertificateProvider: public CertificateProvider {
```

Comment:
This is `NullCertificateProvider` unless you want to implement all the methods. If you intended use it for tests then this shouldn't be registered to envoy binary, just register it in tests.

Reply:
I don't finish it yet, I'd like provide a simple provider for user, just with static certificates configured.